v3/otm_apps.cpp

```diff
@@ -345,7 +344,7 @@ bool otm_cylindrical_flyer()
   in.eps_dot0[body] = eps_dot0;
   in.CFL = 1.0;
   in.use_constant_dt = true;
-  in.constant_dt = hpc::time<double>(1.0e-06);
+  in.constant_dt = hpc::time<double>(1.0e-07);
```

Comment:
@lxmota Have you been able to re-try the critical time step estimate?

Reply:
I haven't yet, but only because I haven't had the time! Brandon and I got the contact working yesterday and got some preliminary results! We think some of the material properties for the hardening law that I was using were off, so we need to rerun with better properties, but we could see the contact, and recoil and some plastic deformation. I'll send out a movie separately.